.github/workflows/rentzila_login_tests.yml

```diff
@@ -0,0 +1,65 @@
+        run: |
+          echo "ADMIN_EMAIL=${{secrets.ADMIN_EMAIL}}" >> .env
+          echo "ADMIN_PASSWORD=${{secrets.ADMIN_PASSWORD}}" >> .env
+          echo "VALID_EMAIL=${{ secrets.VALID_EMAIL}}" >> .env
+          echo "VALID_PASSWORD=${{ secrets.VALID_PASSWORD}}" >> .env
+          echo "VALID_PHONE=${{ secrets.VALID_PHONE}}" >> .env
+          echo "HOMEPAGE_URL=${{ secrets.HOMEPAGE_URL }}" >> .env
+          echo "CORRECT_PHONE_NUMBERS_FULL=${{secrets.VALID_PHONE}}" >> .env
+          echo "CORRECT_PHONE_NUMBERS_WITHOUT_PLUS=${{secrets.CORRECT_PHONE_NUMBERS_WITHOUT_PLUS}}" >> .env
+          echo "CORRECT_PHONE_NUMBERS_WITHOUT_PLUS38=${{secrets.CORRECT_PHONE_NUMBERS_WITHOUT_PLUS38}}" >> .env
+
+          echo "INCORRECT_PHONE_NUMBERS_WITHOUT_COUNTRY_CODE=${{secrets.INCORRECT_PHONE_NUMBERS_WITHOUT_COUNTRY_CODE}}" >> .env
+          echo "INCORRECT_PHONE_NUMBERS_WITHOUTLN=${{secrets.INCORRECT_PHONE_NUMBERS_WITHOUTLN}}" >> .env
+          echo "INCORRECT_PHONE_NUMBERS_WITH_DASHES=${{secrets.INCORRECT_PHONE_NUMBERS_WITH_DASHES}}" >> .env
+          echo "INCORRECT_PHONE_NUMBERS_WITH_SPACES=${{secrets.INCORRECT_PHONE_NUMBERS_WITH_SPACES}}" >> .env
+          echo "INCORRECT_PHONE_NUMBERS_WITH_BRACKETS=${{secrets.INCORRECT_PHONE_NUMBERS_WITH_BRACKETS}}" >> .env
+          echo "INCORRECT_PHONE_NUMBERS_WITHOUT_COUNTRY_CODE_BRACKETS=${{secrets.INCORRECT_PHONE_NUMBERS_WITHOUT_COUNTRY_CODE_BRACKETS}}" >> .env
+          echo "INCORRECT_PHONE_NUMBERS_ELEVEN_DIGITS=${{secrets.INCORRECT_PHONE_NUMBERS_ELEVEN_DIGITS}}" >> .env
+          echo "INCORRECT_PHONE_NUMBERS_OTHER_COUNTRY_CODE=${{secrets.INCORRECT_PHONE_NUMBERS_OTHER_COUNTRY_CODE}}" >> .env
+          echo "INCORRECT_PHONE_NUMBERS_WITHOUT_38=${{secrets.INCORRECT_PHONE_NUMBERS_WITHOUT_38}}" >> .env
+
+          echo "INCORRECT_EMAILS_SPACE=${{secrets.INCORRECT_EMAILS_SPACE}}" >> .env
+          echo "INCORRECT_EMAILS_CYRILLIC=${{secrets.INCORRECT_EMAILS_CYRILLIC}}" >> .env
+          echo "INCORRECT_EMAILS_WITHOUT_AT=${{secrets.INCORRECT_EMAILS_WITHOUT_AT}}" >> .env
+          echo "INCORRECT_EMAILS_WITHOUT_DOT=${{secrets.INCORRECT_EMAILS_WITHOUT_DOT}}" >> .env
+          echo "INCORRECT_EMAILS_WITHOUT_COM=${{secrets.INCORRECT_EMAILS_WITHOUT_COM}}" >> .env
+          echo "INCORRECT_EMAILS_WITHOUT_GMAIL=${{secrets.INCORRECT_EMAILS_WITHOUT_GMAIL}}" >> .env
+          echo "INCORRECT_EMAILS_WITHOUT_AT_GMAIL=${{secrets.INCORRECT_EMAILS_WITHOUT_AT_GMAIL}}" >> .env
+          echo "INCORRECT_EMAILS_WITH_TWO_AT=${{secrets.INCORRECT_EMAILS_WITH_TWO_AT}}" >> .env
+
+          echo "INCORRECT_PASSWORDS_SPACE_AT_END=${{secrets.INCORRECT_PASSWORDS_SPACE_AT_END}}" >> .env
+          echo "INCORRECT_PASSWORDS_SPACE_AT_BEGINNING=${{secrets.INCORRECT_PASSWORDS_SPACE_AT_BEGINNING}}" >> .env
+          echo "INCORRECT_PASSWORDS_NON_EXISTENT=${{secrets.INCORRECT_PASSWORDS_NON_EXISTENT}}" >> .env
+          echo "INCORRECT_PASSWORDS_NO_UPPERCASE=${{secrets.INCORRECT_PASSWORDS_NO_UPPERCASE}}" >> .env
```

Comment:
We do not store invalid test data as env vars. This makes no sense, this data is irrelevant and does not contain any value. 
Only sensitive data (like valid credentials) should be hidden in env vars